tap_trello/streams.py

```diff
@@ -373,8 +373,52 @@ class Cards(AddCustomFields, ChildStream):
     key_properties = ["id"]
     replication_method = "FULL_TABLE"
     parent_class = Boards
-    MAX_API_RESPONSE_SIZE = 20000
-    params = {'limit': 20000, 'customFieldItems': 'true'}
+    MAX_API_RESPONSE_SIZE = 1000
+
+    def get_records(self, format_values, additional_params=None):
+        # Get max_api_response_size from config and set to paramater
+        response_size = self.config.get('max_api_response_size_card')
+        if response_size and int(response_size):
+            self.MAX_API_RESPONSE_SIZE = int(response_size)
+        self.params = {'limit': self.MAX_API_RESPONSE_SIZE, 'customFieldItems': 'true'}
+
+        # Set window_end with current time
+        window_end = utils.strftime(utils.now())
+
+        custom_fields_map, dropdown_options_map = self.build_custom_fields_maps(parent_id_list=format_values)
+        while True:
+
+            # Get records for cards before specified time or card ID
```

Comment:
Can explain in a comment, why we can use card ID instead of time?

Reply:
Yes, I added details and a reference article link for the same.